.github/workflows/PR.yml

```diff
@@ -0,0 +1,36 @@
+name: NIDAQmx package
+
+on: [push]
```

Comment:
In addition to naming this action "PR", it should execute when someone creates or updates a PR:
```
on:
  pull_request:
    branches:
      - main
```

If the tests end up taking too long to run for every PR, we could also create a "CI" action that executes when someone pushes to main. However, I don't think we need this yet.

BTW, this repo still uses "master" not "main".
Cc: @zhindes

Reply:
Yeah, sorry about that. I intend to change it at some point, but no concrete plans at the moment.